docs/configuration.md

```diff
@@ -0,0 +1,140 @@
+              rif: '',
+              fifsAddrRegistrar: '',
+              rskOwner: '',
+            },
+          },
+        };
+    }
+  }
+```
+**Note: We only show the important piece of code here.**
+
+As you can see we have a structure like this:
+
+```json5
+{
+  lumino: {
+    explorer: {
+      endpoint: '',
+    },
+  },
+  notifier: {
+    availableNodes: [],
+  },
+  rns: {
+    contracts: {
+      rns: '',
+      publicResolver: '',
+      multiChainResolver: '',
+      rif: '',
+      fifsAddrRegistrar: '',
+      rskOwner: '',
+    },
+  },
+}
+```
+
+Now we need to fill those empty fields, here is some reference about what
+those fields are:
+
+```json5
+{
+  lumino: {
+    explorer: {
+      endpoint: 'Here you need to put the endpoint url where the Lumino Explorer API is running, for example http://localhost:8080/api/v1',
+    },
+  },
+  notifier: {
+    availableNodes: [
+      // This is an array of notifier endpoints, you need at least 3.
+      'The endpoint for the notifier 1 (where the API is running), for example: http://localhost:8081/',
+      'The endpoint for the notifier 2 (where the API is running), for example: http://localhost:8082/',
+      'The endpoint for the notifier 3 (where the API is running), for example: http://localhost:8083/'
+    ],
+  },
+  rns: {
+    contracts: {
+      // All these values can be retrieved from the RNS Suite deployment summary.
+      rns: 'The RNS Contract Address',
+      publicResolver: 'The Public Resolver Contract Address',
+      multiChainResolver: 'The Multi-chain Resolver Contract Address',
+      rif: 'The RIF Token Address',
+      fifsAddrRegistrar: 'The FIFSAddrRegistrar Contract Address',
+      rskOwner: 'The RSK Owner Contract Address',
+    },
+  },
+}
+```
+
+As you can see we need to setup some endpoints (for Lumino Explorer and Notifiers) and then the RNS Suite Contract Addresses.
+Lumino Hub doesn't need to be here since Lumino Explorer provide that for us in runtime so we are ok.
+
+Here is an example of a local configuration:
+
+```json5
+{
+  lumino: {
+    explorer: {
+      endpoint: 'http://localhost:8080/api/v1',
+    },
+  },
+  notifier: {
+    availableNodes: [
+      'http://localhost:8081/',
+      'http://localhost:8082/',
+      'http://localhost:8083/',
+    ],
+  },
+  rns: {
+    contracts: {
+      rns: '0x83C5541A6c8D2dBAD642f385d8d06Ca9B6C731ee',
+      publicResolver: '0xE0825f57Dd05Ef62FF731c27222A86E104CC4Cad',
+      multiChainResolver: '0x73ec81da0C72DD112e06c09A6ec03B5544d26F05',
+      rif: '0x1Af2844A588759D0DE58abD568ADD96BB8B3B6D8',
+      fifsAddrRegistrar: '0x8921BF2f074b5470c02Cc7473F17282576111591',
+      rskOwner: '0x79bbC6403708C6578B0896bF1d1a91D2BB2AAa1c',
+    },
+  },
+}
+```
+
+**IMPORTANT: These parameters are only initial parameters, the wallet needs to be
+re-installed (removed and installed again) on the browser after we change this, otherwise we will not see any change at all.**
```

Comment:
what about a short section on how to remove the wallet so that the user can re-install it again if needed? (maybe not in this page)

Reply:
That's part of "how to use" and "how to add or remove add-ons" on each browser, i think if you know how to add them, remove the wallet should be clear, also we have 4 supported browsers and that section could be a lot of information, let's just assume that the user is an advanced user, because who uses this for development is not a basic user actually